ngx-tools/coercion/src/date/date-property.spec.ts

```diff
@@ -2,37 +2,37 @@ import { coerceDateProperty } from './date-property';
 
 
 describe(`coerceDateProperty`, () => {
+  const passInDate = new Date('2018-08-08');
 
   test(`should coerce undefined to 0 or default`, () => {
```

Comment:
I would change all these test blocks to say something like `should return the default value when ____`. (that way the message will hold true even if we change the default later)